EngineeringModel/Views/ElementDefinitionBrowser/ElementDefinitionsBrowser.xaml

```diff
@@ -386,6 +387,9 @@
                         <dxg:TreeListColumn FieldName="ModelCode" Header="Model Code" />
                         <dxg:TreeListColumn FieldName="RowType" Header="Row Type" />
                         <dxg:TreeListColumn FieldName="IsObfuscated" Header="IsObfuscated" Visible ="False"/>
+                        <dxg:TreeListColumn Binding="{Binding Category, Converter={StaticResource CategoryListConverter} }"
```

Comment:
Can you please move it up to between formula and modelcode. Also, it would be nice if filtering was allowed (and then per each category. This can be confirmed with @lxatrhea but it would be nice to see this in advanced fltering.

Reply:
@alexatrhea Filtering is allowed on the other Category column that is defined from row 393 and further. 
Category filtering on that other column is done on IsMemberOf and hasCategoryApplied and was implemented last year.